ci-operator/jobs/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main-presubmits.yaml

```diff
@@ -19,6 +19,17 @@ presubmits:
       job-release: "4.14"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-eng-ocp-qe-perfscale-ci-main-aws-4.14-nightly-arm-control-plane-24nodes
+    reporter_config:
```

Comment:
I don't see any configuration to allow us to filter out periodics vs presubmits. So, we must be willing to accept these extra reporter configs. I think it's fine.